src/Home/Home.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import firebase from "firebase/app";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+const Home = () => {
```

Comment:
Isn't this going to be the 'feed' component later on, or are we keeping homepage and add the feed into that?

Reply:
This can be the feed component later on